java/DynamoDBStreamSource/README.md

```diff
@@ -0,0 +1,37 @@
+# DynamoDB Streams Source example
+
+* Flink version: 1.20
+* Flink API: DataStream API
+* Language: Java (11)
+* Flink connectors: DynamoDb Streams Source
+
+
+This example demonstrate how to use Flink DynamoDB Streams source.
+
+This example uses `DynamoDbStreamsSource`.
+
+### Runtime configuration
+
+The application reads the runtime configuration from the Runtime Properties, when running on Amazon Managed Service for Apache Flink,
+or, when running locally, from the [`resources/flink-application-properties-dev.json`](resources/flink-application-properties-dev.json) file located in the resources folder.
+
+All parameters are case-sensitive.
+
+| Group ID        | Key           | Description               | 
+|-----------------|---------------|---------------------------|
+| `InputStream0`  | `stream.arn` | ARN of the input stream.  |
+
+Every parameter in the `InputStream0` group is passed to the DynamoDB Streams consumer.
```

Comment:
"...for example `flink.stream.initpos`..."

Reply:
added